torchbenchmark/__init__.py

```diff
@@ -336,7 +336,6 @@ def set_train(self) -> None:
     def invoke(self) -> None:
         self.worker.run("""
             model.invoke()
-            maybe_sync()
```

Comment:
Sorry I don't understand why you are removing this sync function?

Reply:
The idea is to remove cuda sync at every step.

Reply:
cc @ngimel @ptrblck @csarofeen

Reply:
Sorry I am asking why remove cuda sync at every step? Currently running CUDA sync at the end of every test in the subprocess is our assumption.